craftassist/servermgr/app.py

```diff
@@ -149,54 +126,6 @@ def status():
     return json.dumps({"progress": 100, "ip": ip})
 
 
-@app.route("/matchmaker")
-def matchmaker_home():
-    return flask.render_template("matchmaker.html")
-
-
-@app.route("/matchmaker/launch", methods=["GET", "POST"])
-def matchmaker_launch():
-    SCRIPT = """
-local prefix = 'matchmaker:'
-local wait_key = redis.call('GET', prefix..'wait_key')
-if not wait_key then
-    local wait_key = prefix..math.random()
-    redis.call('SET', prefix..'wait_key', wait_key, 'EX', 600)
-    return cjson.encode({action="launch", wait_key=wait_key})
-else
-    redis.call('DEL', prefix..'wait_key')
-    return cjson.encode({action="wait", wait_key=wait_key})
-end
-    """
-    r = rconn.eval(SCRIPT, numkeys=0)
-    d = json.loads(r)
-    logging.info("matchmaker script returned {}".format(d))
-
-    if d["action"] == "launch":
-        instance_id = launch_instance("craftassist-server")
-        rconn.set(d["wait_key"], instance_id, ex=3600)
-        return flask.redirect("/wait/{}?role=manager".format(urlencode(instance_id)))
-
-    elif d["action"] == "wait":
-        while True:
-            instance_id = rconn.get(d["wait_key"])
-            if instance_id:
-                return flask.redirect("/wait/{}?role=assistant".format(urlencode(instance_id)))
-            else:
-                time.sleep(0.5)
-
-
-@app.route("/survey", methods=["POST"])
```

Comment:
we probably should also arxiv this. At some point, we might want to re-introduce this ?